react/features/chat/components/web/MessageContainer.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+
+import React from 'react';
+
+import AbstractMessageContainer, { type Props }
+    from '../AbstractMessageContainer';
+
+import ChatMessageGroup from './ChatMessageGroup';
+
+/**
+ * Displays all received chat messages, grouped by sender.
+ *
+ * @extends AbstractMessageContainer
+ */
+export default class MessageContainer extends AbstractMessageContainer<Props> {
```

Comment:
Maybe it's better not to implement a parametrized class if you don't need to extend or override the props